### ISSUE TITLE
[FEA] Add quality of life improvements to the Devcontainer

### ISSUE THREAD

**Issue by mdemoret-nv** [nv-morpheus/Morpheus#703]
After #702 is implemented, the devcontainer could be improved with some quality of life features. Most of these are nice to have and could be separated out into different issues if difficult to implement:

- [x] Create a script to install `pytest-kafka` (See [this](https://github.com/mdemoret-nv/Morpheus/blob/mdd_improve-devcontainer/.devcontainer/bin/install-pytest-kafka) for an example taken from the CI runner image)
- [x] Setup the VS Code max file watchers to prevent warnings that there are too many files to watch in the folder.
- [x] Add the necessary X Server bindings to support opening the Morpheus Visualization demos from the devcontainer
- [ ] Setup Git LFS storage property to map back to the host to reduce LFS bandwidth
   - [ ] Requires setting the user's global git config `lfs.storage` property to a directory mounted from the host
- [x] Install .NET to allow the CMake extension to use the language server features. (Example [here](https://github.com/mdemoret-nv/Morpheus/blob/mdd_improve-devcontainer/.devcontainer/devcontainer.json#L72-L97))
- [x] Use docker-compose with devcontainers to streamline setting up the docker network

**Comment by cwharris** [comment]
GIT LFS storage directory is designed to be a sub-directory of the `.git` repository directory which will be mounted in to the container, so modifying `lfs.storage` is not necessary for it to be used by both the host and container.
